projects/ccf-rui/src/app/shared/components/organ-selector/organ-selector.component.theme.scss

```diff
@@ -8,7 +8,7 @@
   .carousel {
     color: mat-color($foreground, text);
     border: 1px solid mat-color($foreground, borders);
-    background-color: mat-color($background, dialog);
+    background-color: white;
```

Comment:
This should refer to a variable in the background (you can add additional variables in the theme files if needed) not be hardcoded!